pystac_client/item_search.py

```diff
@@ -361,88 +371,88 @@ def _format_bbox(value: Optional[BBoxLike]) -> Optional[BBox]:
         return bbox
 
     @staticmethod
-    def _format_datetime(value: Optional[DatetimeLike]) -> Optional[Datetime]:
```

Comment:
pulled these methods out of being inside this one, as it made this method very long.